webui/react/src/components/Table/InteractiveTable.module.scss

```diff
@@ -5,7 +5,7 @@
   }
 }
 .row {
-  height: 60px;
+  height: 48px;
```

Comment:
nit: I guess we're replacing this table anyway, but I think height value isn't doing anything here.

Reply:
the table will be replaced just for exp but yeah probably eventually they will all be replaced?

But doesn't the experiment list table look more compact for you? from 60 to 48

Reply:
it is more compact, i just meant that it's at 49px based on content and reducing it to 48px isn't doing anything